api/src/services/observation-service.ts

```diff
@@ -0,0 +1,39 @@
+import { IDBConnection } from '../database/db';
+import { InsertObservation, ObservationRecord, ObservationRepository, UpdateObservation } from '../repositories/observation-repository';
+import { getLogger } from '../utils/logger';
+import { DBService } from './db-service';
+
+const defaultLog = getLogger('services/observation-service');
+
+export class ObservationService extends DBService {
+  observationRepository: ObservationRepository;
+
+  constructor(connection: IDBConnection) {
+    super(connection);
+    this.observationRepository = new ObservationRepository(connection);
+  }
+
+  /**
+   * TODO
+   *
+   * @param {((Observation | ObservationRecord)[])} observations
+   * @return {*}  {Promise<ObservationRecord[]>}
+   * @memberof ObservationService
+   */
+  async insertUpdateSurveyObservations(observations: (InsertObservation | UpdateObservation)[]): Promise<ObservationRecord[]> {
+    defaultLog.debug({ label: 'insertUpdateObservationRecords' })
```

Comment:
not sure if this debug is needed

Reply:
Agreed, it's just there for testing purposes